ui/app/pages/asset/components/asset-options.js

```diff
@@ -0,0 +1,80 @@
+import React, { useContext, useState } from 'react';
+import PropTypes from 'prop-types';
+
+import { I18nContext } from '../../../contexts/i18n';
+import { Menu, MenuItem } from '../../../components/ui/menu';
+
+const AssetOptions = ({
+  onRemove,
+  onViewEtherscan,
+  onViewAccountDetails,
+  tokenSymbol,
+  isNativeAsset,
+}) => {
+  const t = useContext(I18nContext);
+  const [assetOptionsButtonElement, setAssetOptionsButtonElement] = useState(
+    null,
+  );
+  const [assetOptionsOpen, setAssetOptionsOpen] = useState(false);
+
+  return (
+    <>
+      <button
+        className="fas fa-ellipsis-v asset-options__button"
+        data-testid="asset-options__button"
+        onClick={() => setAssetOptionsOpen(true)}
+        ref={setAssetOptionsButtonElement}
+        title={t('AssetOptions')}
```

Comment:
You changed this translation key which is trying to find a key in app/_locales/en/messages.json  named 'AssetOptions' which doesn't exist. If you do want to change this you'll need to change the key to 'assetOptions' and update 'tokenOptions' to match, as long as that's wanted.

Reply:
@brad-decker ah yes that makes more sense. I had originally changed the key in the locale files but wasn't sure that was the right thing to do given that the translations won't really match. Not sure what the right move is here, but feel like it could make sense to revert this key you've highlighted but leave the other name changes as is?

Reply:
In these cases we'd change en/messages.json `tokenOptions: "Token options"` -> `assetOptions: "Asset options"`, then we'd make sure the code is `t('assetOptions')` and finally run yarn lint:fix which will remove the unused translations in the other files. At some point in the future the translation service will add the new 'assetOptions' translations to other files.

Reply:
The lack of translation here is unfortunate but not the end of the world and aligns more with our other terminology anyways. The tab is the 'assets' tab which includes the tokens that would have previously had the 'Token Options'